src/sensors/bmi323sensor.h

```diff
@@ -0,0 +1,184 @@
+/*
+    SlimeVR Code is placed under the MIT license
+    Copyright (c) 2022 TheDevMinerTV
```

Comment:
I'm not sure this copyright is correct either

Reply:
I fixed the copyright